reference/catalog/products_catalog.v3.yml

```diff
@@ -262,7 +262,7 @@ paths:
                         id: 0
                         product_id: 0
                         length: string
-        required: false
+        required: true
```

Comment:
```suggestion
```

Reply:
We don't typically mark request bodies required or not. I am pretty sure this is good to remove altogether.